packages/esm-patient-registration-app/src/patient-registration/field/field.scss

```diff
@@ -125,3 +137,7 @@
   line-height: 1.34;
   margin: 0.25rem 0 0;
 }
+
+.numberInput {
+  width: 13.125rem;
```

Comment:
giving the number inputs a small size

Reply:
For now, let it be either 50% of the form width or full width

Reply:
This update is majorly to reduce the width of the inputs not to be taking up 100% of the width like the [design](https://app.zeplin.io/project/60d59321e8100b0324762e05/screen/60d59b087969aa11e8f4c37c) says. The full input width `(300px)` looks like as below on desktop.
![image](https://github.com/openmrs/openmrs-esm-patient-management/assets/104269786/c1c3db40-0c08-49af-8c35-8568b8bf1fb2)